src/node_file.h

```diff
@@ -319,7 +411,7 @@ class FileHandle : public AsyncWrap, public StreamBase {
       ref_.Reset(env->isolate(), ref);
     }
 
-    ~CloseReq() {
+    virtual ~CloseReq() {
```

Comment:
Wouldn’t `override` do the job here? It’s what you used in other PRs, I think.

Reply:
Yep.